### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by j-manio**
Please check every work in prac_05. @JocoBuen

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    else:
+        print("Invalid short state")
+
```
Comment (JocoBuen) [inline prac_05/state_names.py:17]: no need for space here

File: prac_05/hex_colours.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+print(COLOR_TO_CODE)
+
+colour_choice = input('Colour name: ').lower()
+while colour_choice != "":
+    if colour_choice in COLOR_TO_CODE:
+        print(f"{COLOR_TO_CODE.get(colour_choice)}")
+    else:
+        print("Invalid name!")
+    colour_choice = input('Colour name: ').lower()
```
Comment (JocoBuen) [inline prac_05/hex_colours.py:13]: always check the format of your program (ctrl + alt+ L) is a helpful shortcut

File: prac_05/word_occurence.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+max_length = max(len(word) for word in words)
+for word in words:
+    print(f'{word:{max_length}} : {word_to_count[word]}')
+
+
+
+
+
+
```
Comment (JocoBuen) [inline prac_05/word_occurence.py:21]: check your format every after new changes and after finishing your program.

File: prac_05/word_occurence.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+word_to_count = {}
```
Comment (JocoBuen) [inline prac_05/word_occurence.py:1]: You forgot to put the time of your work.

File: prac_05/word_occurence.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+get_text = input('Text: ')
+words = get_text.split()
+for word in words:
```
Comment (JocoBuen) [inline prac_05/word_occurence.py:5]: Use function for SRP.

File: prac_01/emails.py [hunk 6]
```diff
@@ -0,0 +1,25 @@
+def main():
```
Comment (JocoBuen) [inline prac_01/emails.py:1]: No estimation time again for this program.

File: prac_05/wimbledon.py [hunk 7]
```diff
@@ -0,0 +1,46 @@
+"""
+Wimbledon
```
Comment (JocoBuen) [inline prac_05/wimbledon.py:2]: No estimated time for this program.

File: prac_05/wimbledon.py [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    records = []
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()  # Remove header
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+main()
```
Comment (JocoBuen) [inline prac_05/wimbledon.py:46]: Just commit after every milestone or after finishing one program. Avoid committing empty files.

File: prac_05/wimbledon.csv [hunk 9]
```diff
@@ -0,0 +1,55 @@
+Year,Country,Champion,Country,Runner-up,Score in the final
```
Comment (JocoBuen) [inline prac_05/wimbledon.csv:1]: CSV file does not need to be pushed or included.

**Review by JocoBuen — COMMENTED** [review]
Don't forget to commit to every milestone of the practicals. 
Don't forget to check the format of the program every time.

**Comment by j-manio** [comment]
Thank you very much for reviewing my work and my code. I'll keep them in mind on the next prac.

**Pull request merged**